README.md

```diff
@@ -298,7 +310,7 @@ on `:61678/metrics`.
 
 Type: String
 
-Default: `eni`
+Default: `veth`
```

Comment:
unrelated change? Also, I remember seeing some pushback on the whole eni -> veth rename patch from Calico. Would be good to undo this particular change I think.